NEWS.md

```diff
@@ -1,2 +1,7 @@
-## v1.6.0
-- TBD
\ No newline at end of file
+## v2.0.0
+### Features
+- Create New User
+- Get User Detail
+- Check if user exists
+- Set Patron PIN
+- Verify Patron PIN
```

Comment:
I created some tickets that were done prior to this release in the JIRA. Maybe we should change this text to better reflect the community practice of releases. It now could be changed to the following just to get us going in the right way. Probably these features should be covered in the readme rather than the release news.
```
* [Update commons-lang3 version to 3.17.0](https://folio-org.atlassian.net/browse/EDGEUSERS-3)
* [Review/Resolve identified CVEs in edge-users](https://folio-org.atlassian.net/browse/EDGEUSERS-2)
```